packages/opentelemetry-context-async-hooks/package.json

```diff
@@ -40,17 +40,9 @@
     "access": "public"
   },
   "devDependencies": {
-    "@types/mocha": "8.2.0",
-    "@types/node": "14.14.20",
-    "@types/shimmer": "1.0.1",
     "codecov": "3.8.1",
-    "gts": "3.1.0",
-    "mocha": "7.2.0",
     "nyc": "15.1.0",
-    "rimraf": "3.0.2",
-    "ts-mocha": "8.0.0",
-    "ts-node": "9.1.1",
-    "typescript": "4.1.3"
+    "rimraf": "3.0.2"
```

Comment:
don't think rimraf is needed here.

Reply:
it was included in a lot of packages that didn't need it. removed them all